pkg/apis/extensions/v1alpha1/types_bastion.go

```diff
@@ -0,0 +1,119 @@
+// Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1alpha1
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ Object = (*Bastion)(nil)
+
+// BastionResource is a constant for the name of the Bastion resource.
+const BastionResource = "Bastion"
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:resource:scope=Namespaced,path=bastions,singular=bastion
+// +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name=IP,JSONPath=".spec.bastionIP",type=string,description="The public IP address of the temporary bastion host"
+// +kubebuilder:printcolumn:name=State,JSONPath=".status.state",type=string,description="The bastion's status."
+// +kubebuilder:printcolumn:name=Age,JSONPath=".metadata.creationTimestamp",type=date,description="creation timestamp"
```

Comment:
I think that's not needed?

Reply:
All of them or just the Age column? With Age I was also unsure, but I copied what we had for other CRDs ^^

Reply:
Hm, I missed that we are indeed using these `kubebuilder` tags also for other resources. It seems fine for now then, resolving the comment.